artemis-website/package-lock.json

```diff
@@ -3676,7 +3676,7 @@
       "dev": true,
       "requires": {
         "graceful-fs": "^4.1.5",
-        "minimist": "^1.2.6",
+        "minimist": "^1.2.0",
```

Comment:
Seems like the changes in this file should go in separately if needed, they seem unrelated.

Reply:
This was a mistake. Reverted.